Gordon360/Models/CCT/CUSTOM_PROFILE.cs

```diff
@@ -25,5 +25,7 @@ public partial class CUSTOM_PROFILE
         public string linkedin { get; set; }
         [Unicode(false)]
         public string handshake { get; set; }
+        [Unicode(false)]
+        public string calendar { get; set; }
```

Comment:
This is not part of your changes, usually best practice is to have a separate branch & PR for database changes (or just merge in develop ;) )

I'm not going to die on this hill though.

Reply:
We need to merge in develop! Right now I can't keep track of the updates from EFCore as we have a bunch of these updates scattered everywhere.